source/2_platform/rex_windows/src/gui_application.cpp

```diff
@@ -64,28 +63,17 @@ namespace rex
 
       void loop()
       {
-        FrameInfo info = {World::get_delta_time(), World::get_frames_per_seconds()};
-
+        const FrameInfo info;
         on_update(info);
 
         window->update();
-        world.update();
 
         is_running = !is_marked_for_destroy;
 
         //
         // Cap framerate to "max_fps".
         // Safe resources of the machine we are running on.
         //
-        const rsl::chrono::milliseconds actual_time(static_cast<int64>(rsl::lrint(1000.0f / static_cast<f32>(world.get_frames_per_seconds().get()))));
```

Comment:
Why is this removed, we do not want to fly through the app. We should cap it on a specific framerate IMO.

Reply:
there's no world anymore, we'll put it back when we figure out fps